includes/functions.php

```diff
@@ -2025,15 +2027,29 @@ function device_is_up($device, $record_perf = false)
         if ($device['snmp_disable'] || isSNMPable($device)) {
             $response['status'] = '1';
             $response['status_reason'] = '';
-        } else {
+        }
+        else {
             echo 'SNMP Unreachable';
             $response['status'] = '0';
-            $response['status_reason'] = 'snmp';
+            if ($maintenance && $mode) {
+                // Scheduled maintenance, device not responding to snmp
+                $response['status_reason'] = 'snmp (maintenance)';
```

Comment:
have you rechecked the system, to avoid conflicts through this new state?

Reply:
I did some testing and could not find any issues yet

Reply:
i grepped through system for status_reason and found many possible upcoming problems. A thinkover would be useful in my eyes.

Reply:
I will try another way to detect this state change without modifying the status_reason, WIP